Sign Up Page/signup.css

```diff
@@ -0,0 +1,98 @@
+body {
+  --gradient: linear-gradient(
+    60deg,
+    #953773,
+    #8374ab,
+    #8984c7,
+    #a4b7dc,
+    #be96cd
+  );
+  margin: 0;
+  height: 100vh;
+  font-size: 16px;
+}
+
+@media (max-width: 425px) {
+  body {
+    background-color: hsl(152, 76%, 67%);
+    display: flex;
+    align-items: center;
+    justify-content: center;
+  }
+
+  .maincon {
+    border-style: 1px solid;
+    background-color: hsl(180, 95%, 33%);
+    text-align: center;
+  }
+
+  .namecon,
+  .mailcon,
+  .passcon,
+  .rpasscon,
+  .signb {
+    margin: 1em;
+  }
+
+  a {
+    text-decoration: none;
+  }
+
+  #user_name,
+  #email,
+  #password,
+  #confirm_password {
+    border-radius: 8px;
+    width: 20em;
+    height: 2em;
+  }
+
+  #submitbtn {
+    border-radius: 8px;
+    width: 275px;
+    height: 31px;
+  }
+}
+
+@media (min-device-width: 426px) and (max-width: 2560px) {
+  body {
+    background-image: var(--gradient);
+    background-size: 300%;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+  }
+
+  .maincon {
+    border-style: solid;
+    background-color: hsl(180, 95%, 33%);
+    text-align: center;
+  }
+
+  .namecon,
+  .mailcon,
+  .passcon,
+  .rpasscon,
+  .signb {
+    margin: 1em;
+  }
+
+  a {
+    text-decoration: none;
+  }
+
+  #user_name,
+  #email,
+  #password,
+  #confirm_password {
+    border-radius: 8px;
+    width: 20em;
+    height: 2em;
+  }
```

Comment:
You can add a single class to these elements instead of accessing them via IDEs

Reply:
I changed it in recent commits.